content/SCALE/SCALETutorials/SystemSettings/Advanced/ManageCronJobsSCALE.md

```diff
@@ -17,25 +17,35 @@ The **Cron Jobs** widget on the **System > Advanced** screen displays **No Cron
 
 ![AdvancedSettingsCronJobWidget](/images/SCALE/22.02/AdvancedSettingsCronJobWidget.png "SCALE Advanced Settings Cron Job Widget") 
 
-Click **Add** to open the **Add Cron Job** configuration screen to create a new cron job, or if you want to modify an existing job, click anywhere on the item to open the **Edit Cron Jobs** configuration screen populated with the settings for that cron job. 
+Click **Add** to open the **Add Cron Job** configuration screen and create a new cron job. If you want to modify an existing Cron Job, click anywhere on the item to open the **Edit Cron Jobs** configuration screen populated with the settings for that cron job. 
 The **Add Cron Job** and **Edit Cron Job** configuration screens display the same settings. 
 
 ![AddCronJobsScreen](/images/SCALE/22.02/AddCronJobScreen.png "SCALE Console Settings Screen")
 
 Enter a description for the cron job.
 
-Next, enter the full path to the command or script to run in **Command**. For example, a command string to create a list of users on the system and write that list to a file enter `cat /etc/passwd > users_$(date +%F).txt`.
+Next, enter the full path to the command or script to run in **Command**. For example, for a command string to create a list of users on the system and write that list to a file, enter `cat /etc/passwd > users_$(date +%F).txt`.
 
 Select a user account to run the command from the **Run As User** dropdown list. The user must have permissions allowing them to run the command or script. 
 
 Select a schedule preset or choose **Custom** to open the advanced scheduler. 
-Note that an in-progress cron task postpones any later scheduled instance of the same task until the running task is complete. 
+An in-progress cron task postpones any later scheduled instances of the task until the one already running completes. 
+
+{{< expand "Cron Job Schedule Format" "v" >}}
+Cron Job schedules use six asterisks that represent minutes, hours, days of the month, days of the week, and months in that order.
+
+For example, a schedule of *1 1 1 * sat,sun* would run at 01:01 AM, on day 1 of the month, and only on Saturday and Sunday.
+
+{{< hint info >}}
```

Comment:
do we really need the hint admonition box for this text? It just further clarifies what they need to do so a simple hint really isn't necessary, Thoughts?